keyboards/clueboard/60/60.h

```diff
@@ -45,17 +45,17 @@
  * 3 mods on either side of the spacebar.
  */
 #define LAYOUT_aek(\
-    k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c,   k0e, \
-     k10,   k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c,   k1e, \
-     k20,    k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b,    k2e, \
-       k30,   k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b,      k3c, \
+    k00, k01, k02, k03, k04, k05, k06, k07, k08, k09,  k0b, k0a, k0c,   k0e, \
+     k10,   k11, k12, k13, k14, k15, k16, k17, k18, k19, k1b,k1a,  k1c,   k1e, \
```

Comment:
@skullydazed any issues with this change?

Reply:
Yes, I don't understand why this change is here. I'm looking at the pcb design and the original file should be correct?

Reply:
well, that's why I tagged you. 

>revised my keymap, and fixed a glitch in the AEK matrix that swapped the keys in the p and [{ columns.

As reported by @lbibass.  As I don't have the board, I can't even begin to confirm it.

Reply:
yeah, I wasn't sure what was going on, but when I flashed it, those two rows were flipped. I am not sure what caused it, but doing what I did fixed it. Do you mind testing it on your end? I can't think of anything on my end that would have caused that, but I could be wrong.

Reply:
@skullydazed did you ever get a chance to confirm this? Would be great to get this PR modified or merged, considering the lifetime.